Files/Constants.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Files
+{
+    public static class Constants
+    {
+        public static class Browser
+        {
+            public static class GridViewBrowser
+            {
+                public const int GRIDVIEW_SIZE_LARGE = 220;
```

Comment:
Fix naming, please. `GridViewSizeLarge`

Reply:
I intentionally named it with uppercase letters. The constant `const` variables should be distinguishable from other non-constants. We should update the naming scheme for constant values

Reply:
@d2dyno1 Do not use underscores (_) in between words in variables. ([Source](https://www.c-sharpcorner.com/blogs/naming-conventions-in-c-sharpnet))
[Code style](https://github.com/files-community/Files/wiki/Code-style)

Reply:
The uppercase with `_` is a C/C++ naming convention which I think is superior to the provided sources above. It removes any misconceptions between variables and `const` variables. Obviously, by default constants in c# are PascalCase but ignoring that, we would remove any naming ambiguities.

Reply:
If you use C# code in your PR, you have to follow C# rules and our local code style.

Reply:
Too much PascalCases are making the code confusing and I'm sure uppercase constant is not a thing that any developer would find new

Reply:
@yaichenbaum what's your word on that?

Reply:
@d2dyno1 The goal is to have a consistent codebase, this is an area where everyone has their own preference and habits, therefore it's something we discussed a while back and this is what we settled on.

Reply:
As you wish

Reply:
@yaichenbaum Sorry, but it should be the convention to have all constant values as UPPERCASE_NAMES.

Reply:
There must be a way to easily distinguish between normal global variables and constants, and I've always heard the convention is to have constants declared as uppercase.

Reply:
> @yaichenbaum Sorry, but it should be the convention to have all constant values as UPPERCASE_NAMES.

In that case, can you update the Wiki page because right now it doesn't specify that.

Reply:
@yaichenbaum Actually, scratch that. Uppercase isn't consistent with StyleCop and C# specifically. This is an area where C language and C# diverge.